uk.co.compucorp.civicrm.hrleaveandabsences/templates/CRM/HRLeaveAndAbsences/Page/AbsenceTab.tpl

```diff
@@ -1,10 +1,53 @@
 <section data-leave-absences-absence-tab>
-  <ui-view></ui-view>
+  <div class="chr_absence-tab-page">
```

Comment:
This code is not meant to stay here. Please copy this into a separate .html file in views/absence-tab folder(like My Leave and Manager leave) and refer that using php here. Also leave a note in the PR that the linking should be removed when module setup is done.